project-codeflare/codeflare-operator
# Support package refactor

# Issue link
Closes https://github.com/project-codeflare/codeflare-operator/issues/237

# What changes have been made
The support package has been removed and moved to a separate repo [here](https://github.com/project-codeflare/codeflare-common). Relevant imports have been updated.

# Verification steps
- Checkout this PR
- [Run the e2e tests](https://github.com/project-codeflare/codeflare-operator#testing) and confirm that they work as expected. 

## Checks
- Testing Strategy
